### DISCUSSION TITLE
LSP4IJ 0.0.1 is still not released?

### DISCUSSION THREAD

**Discussion by seachicken** [redhat-developer/lsp4ij#303] [Q&A] [answered]
Hello, I have a question. LSP4IJ [0.0.1](https://github.com/redhat-developer/lsp4ij/releases/tag/0.0.1) was recently released, but it seems to have not been released in the marketplace. Does this mean it is still under review there?

<img width="1331" alt="スクリーンショット 2024-05-24 22 06 29" src="https://github.com/redhat-developer/lsp4ij/assets/5178598/1d18089c-e048-41af-9596-d2964efe6e7f">

https://plugins.jetbrains.com/plugin/23257-lsp4ij/versions

**Comment by angelozerr** [comment, marked as answer]
We published the release in marketplace but Jetbrains must validate the lsp4ij plugin.

**Reply by seachicken** [reply]
Nice, thank you for replying
